robolectric-shadows/shadows-core/src/main/resources/android/hardware/display/RoboDisplayManager.java.vm

```diff
@@ -15,7 +15,7 @@ import android.view.Surface;
 
 public class RoboDisplayManager implements IDisplayManager {
 
-#if ($apiLevel >= 21)
+#if ($apiLevel >= 19)
   // API21+
```

Comment:
Ran into some git trouble...

Made the change
Commit
Squash the changes with the original
git push origin fix_broken_robo_display_manager

This change didn't update, but when I went to my own fork, I could see the pull request there, which I made below:-

https://github.com/robolectric/robolectric/pull/1586

I guess you can merge this one, but not sure why it didn't update the original.... thoughts?